utils/wasm-proc/src/main.rs

```diff
@@ -188,14 +188,17 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         }
 
         let file = PathBuf::from(file);
-        let res = gear_wasm_builder::optimize::optimize_wasm(file.clone(), "s", true)?;
-
-        log::info!(
-            "wasm-opt: {} {} Kb -> {} Kb",
-            res.dest_wasm.display(),
-            res.original_size,
-            res.optimized_size
-        );
+
+        #[cfg(feature = "wasm-opt")]
```

Comment:
that is unnecessary imho since `gear_wasm_builder` is able to optimize in both cases.

Reply:
I'd use `wasm-opt` in `wasm-proc` by default as it is the once-building tool.

Reply:
> wondering why we annotated these lines before

I guess it was done to prevent the developer from installing `binaryen`. But if we use the `wasm-opt` crate by default in `wasm-proc`, we'll solve the problem.

Reply:
nope it is due to the issue with the new binaryen - https://github.com/gear-tech/gear/pull/1972/files#r1045663991

The issue relates to signing-extensions whatsoever, @clearloop found the corresponding flags for wasm-opt library:  https://github.com/gear-tech/gear/commit/52e4cc5b95efddbd37c4320f97a208639f94df1e#diff-4d80c505a5a2697acc24e5218d2722d7c0148c70a7ac164fb5224ad9d0ed51d7R185

Reply:
So frankly speaking we also need to set corresponding flag for the `wasm-opt` binary